frame/uniques/src/impl_locker.rs

```diff
@@ -0,0 +1,23 @@
+#![allow(clippy::too_many_arguments)]
```

Comment:
need file header

Reply:
Did I update correctly or do I need to create a `locker.rs` for runtime traits? I am not sure.